base/doc/ltnews41.tex

```diff
@@ -405,6 +405,17 @@ \subsection{Prevent \texttt{cmd} hook from defining an undefined command}
 %
 \githubissue{1591}
 
+\subsection{Process global options once per package}
+
+In 2022, we introduced key--value (keyval) option processing in the
+kernel~\cite{41:ltnews35}. This also added the idea that keys could have scope:
+load-only, preamble-only and general use. However, we overlooked that an option
+given globally (in the optional argument to \cs{documentclass}) would be
+repeatedly processed and could therefore lead to spurious warnings. This has
```

Comment:
The, imho, much bigger issue than these spurious warnings is that `load` options are applied just once while every other option is applied multiple times (potentially overwriting any explicitly set option for this package). But otherwise the wording is fine. This is just a note, not a request for change.